### ISSUE TITLE
Check routine reported an error that doesn't exist

### ISSUE THREAD

**Issue by WGroleau** [fisharebest/webtrees#4788]
I added several handmade records to a GEDCOM and imported.  In the image, it is on the left.  The first two errors were as stated but the third seems to be incorrect.  v. 2.1.16
![temp](https://user-images.githubusercontent.com/5910274/225831865-6485e001-24dd-42bd-8e8f-dfed5ac6b4ae.jpg)

**Comment by fisharebest** [comment]
Does this involve a pending change?

Can you create a minimal GEDCOM file that demonstrates this?

**Comment by WGroleau** [comment]
I'll look at the other errors, but since it didn't happen the second time on the same unchanged record, it might be a futile effort.  The first two errors were corrected but the third was untouched when I did the second import.  The corrected one were not in the report after, and neither was the third though those three records had not been touched.  If the glitch occurs again, I can re-open and try to come up with something, but for now, I don't see what anyone can do with an unchanged record getting a different result.  Doesn't seem possible for other records **earlier** in the file, or **different** rows in the table to affect this one.  For now, I'll dub it "not reproducible"
